testing/time.ts

```diff
@@ -288,7 +264,7 @@ export class FakeTime {
     start?: number | string | Date | null,
     options?: FakeTimeOptions,
   ) {
-    if (time) throw new TimeError("Time is already faked");
+    if (time) throw new TypeError("Time is already faked");
```

Comment:
Note: made to be similar to when a constant tried to get re-assigned ([ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Invalid_const_assignment)).

Reply:
Ditto. This feels like error about the state, not the type. I would use 'Error' if we need to use native error types.

Reply:
Fair point. Changed to `Error`.